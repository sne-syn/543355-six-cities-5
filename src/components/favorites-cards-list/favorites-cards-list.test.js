import React from 'react';
import renderer from 'react-test-renderer';
import FavoritesCardsList from './favorites-cards-list';

it(`FavoritesCardsList is rendered correctly`, () => {
  const tree = renderer.create((
    <FavoritesCardsList
      offers={[]}
      city={`Paris`}
    />
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
