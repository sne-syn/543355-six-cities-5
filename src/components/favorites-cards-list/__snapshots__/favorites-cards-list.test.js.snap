// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesCardsList is rendered correctly 1`] = `
<div
  className="favorites__places"
/>
`;
